Dockerfile

```diff
@@ -8,10 +8,14 @@ RUN apt-get -y update && \
     apt-get -y autoremove
 
 # Copy the latest version of the bot 
-COPY ./discord_quote /discord_quote
+COPY ./src /src
+COPY ./bin /bin
```

Comment:
You're probably just mirroring what we did before, but any idea why we need to do this step?